contrib/build-rpm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$(which rpmbuild)" == "" ];
+then
+  echo "To build an rpm the tool rpmbuild needs to be installed first"
+  exit -1
+fi 
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -z $1 ];
+then
+  echo "Usage $0 \"release\""
+  exit -1
+fi
+
+cd ${DIR}/..
+
+rm google-authenticator-*.tar.gz
```

Comment:
this should be more specific about exactly which file to delete. deleting too many files would be surprising to user, so worth the extra effort.